influxql/select.go

```diff
@@ -822,15 +822,38 @@ func buildTransformIterator(lhs Iterator, rhs Iterator, op Token, ic IteratorCre
 			left:  newBufIntegerIterator(left),
 			right: newBufIntegerIterator(right),
 			fn: func(a *IntegerPoint, b *IntegerPoint) *FloatPoint {
+				if a == nil && b == nil {
+					return nil
+				} else if a == nil {
+					return &FloatPoint{
+						Name:  b.Name,
+						Tags:  b.Tags,
+						Time:  b.Time,
+						Aux:   b.Aux,
+						Value: float64(0),
```

Comment:
I just noticed this. You can just remove this and Go will use the default value.